docs/UsersGuide.asciidoc

```diff
@@ -1133,10 +1133,22 @@ how to configure the limits. Cleanup-related parameter for the REST API can
 be found in the <<UsersGuide.asciidoc#_asset_handling,Asset handling>> section under
 <<UsersGuide.asciidoc#_use_of_the_rest_api,Use of the REST API>>.
 
-==== Cleanup strategy ====
+==== Timers and triggers ====
+
+Cleanup can be triggered in different ways. One option is to use `minion_task_triggers`
+and specify tasks via `on_job_done`. Another way to do that is to use the systemd timers
+`openqa-enqueue-asset-cleanup` and `openqa-enqueue-result-cleanup` to periodically
+run those tasks. Both can be used separately or in combination.
+
+The relevant minion tasks are:
```

Comment:
```suggestion
The relevant Minion tasks are:
```

Reply:
We don't usually spell this uppercase, c.f. `the GRU minion job dashboard` and `umber of parallel cache minion workers, defaults to 5`

Reply:
Well, no strong opinion if you don't mind the inconsistency.